config/environment.js

```diff
@@ -34,19 +34,64 @@ module.exports = function(environment) {
         },
         Reviews: {
             defaultProvider: 'osf',
-        },
-        providers: [
-            // OSF must be the first provider
-            {
-                id: 'osf',
-                logoSharing: {
-                    path: '/assets/img/provider_logos/osf-dark.png',
-                    type: 'image/png',
-                    width: 363,
-                    height: 242
+            providers: [
+                // OSF must be the first provider
```

Comment:
because of line 4 in [ routes/provider.js](https://github.com/CenterForOpenScience/ember-osf-reviews/pull/2/files/1908a8109788fb5618db924e1d2e61798c2aeb47#diff-b4af49876da16fe17ebc9b388cd01841)
``
const providers = config.Reviews.providers.slice(1);
``